src/mahoji/commands/smelt.ts

```diff
@@ -120,8 +117,25 @@ export const smeltingCommand: OSBMahojiCommand = {
 				timeToSmithSingleBar *= 1.075;
 				boosts.push('-7.5% penalty for not having graceful equipped.');
 			}
+			if (user.hasItemEquippedAnywhere('Smithing master cape')) {
+				timeToSmithSingleBar /= 2;
+				boosts.push('2x boost for Smithing master cape');
+			}
+			if (user.hasItemEquippedAnywhere(itemID('Dwarven gauntlets')) && bar.id !== itemID('Gold bar')) {
```

Comment:
This is wrong also. Dwarven gauntlets are supposed to give Gold the 2x boost, except when blast furnace. (smelt.ts rev: b7c0e151 )

Reply:
Reviewed the blastfurnace.ts file and this looks to match, this code is under the header if (blast_furnace) so looks and tests good to me.